tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/PostValidator.java

```diff
@@ -306,6 +306,10 @@ private Collection<? extends FieldError> checkFunding(PostDTO postDto) {
             && !postFundingDto.getEndDate().isAfter(postFundingDto.getStartDate())) {
           fieldErrors.add(new FieldError(POST_DTO_NAME, "fundings",
               "Post funding end date must not be equal to or before start date"));
+        } else if (postFundingDto.getFundingType() == null) {
+          fieldErrors.add(
+              new FieldError(POST_DTO_NAME, "fundings",
+              "Post Funding must have a funding type"));
```

Comment:
I'm a little wondering if this validation in PostValidator is required as Spring takes it over to validate postFunding NotEmpty in the dto.

Reply:
Sorry are you referring to line 300 where we check for empty set? We still need to check each individual post funding for a funding type right?

Reply:
Yes, I did misunderstood the ACs.